lib/ds/arrayutils.flow

```diff
@@ -240,7 +240,11 @@ foldiAsync2(xs : [?], i : int, acc : ??, fn : (int, ??, ?, (??) -> void, (???) -
 		onOK(acc)
 	} else {
 		fn(i, acc, xs[i], \acc2 -> {
-			foldiAsync2(xs, i + 1, acc2, fn, onOK, onError)
+			if (i % 64 == 63) {
+				deferred(\ -> foldiAsync2(xs, i + 1, acc2, fn, onOK, onError));
+			} else {
+				foldiAsync2(xs, i + 1, acc2, fn, onOK, onError);
```

Comment:
make recCall = \ -> foldiAsync2(xs, i + 1, acc2, fn, onOK, onError);

But what is the point? This "magic" definitely lacks some comment.

Reply:
timer breaks the chains of call, it starts a new chain